src/pages/AllMoviesPage.vue

```diff
@@ -1,19 +1,82 @@
 <template>
-  <div>
-    <h1>All Movies Page</h1>
-    <MovieList />
+  <div class="page-title">
```

Comment:
I think we might want to change the class name for this one - it's more of a page on it's own so something like `movies-page` is more accurate.
(I wouldn't expect the page-title to also have things like movies or filters inside 😄)